infra/production/invoke-oci-production-command.sh

```
@@ -0,0 +1,127 @@
+
+OPERATION=""
+TARGET_SHA=""
+COMPARTMENT_ID=""
+INSTANCE_ID=""
+REGION=""
+APPROVED_CONTRACT_FROM_SHA=""
+APPROVED_CONTROL_SHA=""
+APPROVED_MIGRATION_TARGET_SHA=""
+COMMAND_ID=""
+TEMP_DIR=""
+MAX_OUTPUT_BYTES=12288
```

Shellcheck (0.11.0)

[warning] MAX_OUTPUT_BYTES appears unused. Verify use (or export if used externally).

(SC2034)